kickoff/static/model/release.js

```diff
@@ -1,37 +1,79 @@
+// Please also update VALID_VERSION_PATTERN in forms.py
+// Changing the bracket orders may require to update INDEXES and TYPE_INDEXES
+// TODO: Read the pattern from a centralized file
+var VALID_VERSION_PATTERN_STRING  = '^(\\d+)\\.(' + // Major version number
```

Comment:
From @sylvestre in [this "outdated" comment](https://github.com/mozilla-releng/ship-it/pull/114/files/acd88ee4f32cce2c01894bda260972fcc69a8f28#r75752029):

> Oh, this is duplicating code... What about creating a library for this? I am sure we would have other places to use it.

I agree.  Like said [in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1297011#c5), I'd prefer to handle that in a follow. What do you think of it?